statsrelay/src/backend_client/mod.rs

```diff
@@ -0,0 +1,18 @@
+use tokio::{sync::mpsc, task::JoinHandle};
+
+use crate::statsd_proto::Pdu;
+
+pub mod prom_client;
+pub mod statsd_client;
+
+pub trait BackendClient {
+    fn sender(&self) -> mpsc::Sender<Pdu>;
```

Comment:
Would it be worth factoring this as `send(&self, pdu: Pdu)` rather than an accessor for the sender, to avoid leaking the kind of sender it is through the abstraction?